app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_create :increment_like_count
+
+  def increment_like_count
+    post.increment!(:likes_count)
```

Comment:
- [ ] Kindly fix your error here. What you have in the schema is `likes_counter`, not `likes_count`